Yoast/wordpress-seo
# Implemented Sidebar container

## Summary

This PR can be summarized in the following changelog entry:

* Adds a container deal with the displaying of the SEO and Readability analysis based on user preferences.

## Relevant technical choices:

* @atimmer and I decided to use the current implementation to determine the user preferences based on localized scripts as this is a proven solution and the quickest way to move forward.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and ensure you've properly run `yarn upgrade yoast-components`.
* Ensure you have all the necessary flags set to enable Gutenberg and sidebar support.
* Run `yarn start`.
* Add a new post and enable the Yoast Sidebar. See that the labels 'Readability analysis' and 'SEO analysis' show up.
* Go to SEO -> General -> Features  and disable SEO analysis.
* Go back to your post and refresh. Notice that the SEO analysis label has disappeared from the sidebar.
* Go back to SEO -> General -> Features and also disable the Readability analysis.
* Refresh your post again and see that the sidebar does not contain either of the labels.

## Quality assurance


Fixes #10310 
Fixes https://github.com/Yoast/yoast-components/issues/452
